internal/transform/trustline.go

```diff
@@ -61,7 +61,7 @@ func TransformTrustline(ledgerChange ingest.Change, header xdr.LedgerHeaderHisto
 	transformedTrustline := TrustlineOutput{
 		LedgerKey:          outputLedgerKey,
 		AccountID:          outputAccountID,
-		AssetType:          int32(asset.Type),
+		AssetType:          asset.Type.String(),
```

Comment:
@chowbao I am seeing values like `AssetTypeAssetTypePoolShare` in the tables
![Screenshot 2024-07-19 at 10 26 15 AM](https://github.com/user-attachments/assets/d9a26b2e-9f71-40e4-aa10-c4ebe7fa7b93)

Should it rather be `AssetTypePoolShare` ?

Reply:
It should match how `history_assets` does it which I think removes `AssetTypeAssetType`.

You might need to define a new function to make the change or look for an existing if one exists